src/components/forecast/ForecastDetail.js

```diff
@@ -0,0 +1,110 @@
+import { useParams } from 'react-router-dom';
+import { useEffect, useState } from 'react';
+import { useSelector } from 'react-redux';
+import {
+  BsFillCloudMoonFill,
+  BsFillCloudSunFill,
+  BsFillSunFill,
+  BsMoonFill,
+  BsCloudSnowFill,
+  BsCloudRainHeavyFill,
+} from 'react-icons/bs';
+import classes from './ForecastDetail.module.css';
+
+const apiKey = process.env.REACT_APP_KEY;
+const ForecastDetail = () => {
+  const forecast = useSelector((state) => state.forecast.list);
+  const param = useParams();
+  const [data, setData] = useState(null);
+  useEffect(() => {
+    const getForecast = async () => {
+      const response = await fetch(
+        `http://dataservice.accuweather.com/forecasts/v1/daily/1day/${param.cityid}?apikey=${apiKey}%20&details=true&metric=true`,
```

Comment:
- [ ] You completed the deployment of your app successfully. However, when I access the link, I can see the page; however, it is blank. It would be appreciated if you could ensure the data display matches what was tested in the localhost.